ts/store/reducers/backendStatus/__tests__/remoteConfig.test.ts

```diff
@@ -248,3 +249,39 @@ describe("landingScreenBannerOrderSelector", () => {
     });
   }
 });
+
+describe("pnMessageServiceIdSelector", () => {
+  const someState = {
+    remoteConfig: O.some({
+      pn: {
+        notificationServiceId: "NOTIF_SID"
+      }
+    })
+  } as GlobalState;
+  const noneState = {
+    remoteConfig: O.some({
+      pn: {}
+    })
+  } as GlobalState;
+  const emptyStringState = {
+    remoteConfig: O.some({
+      pn: { notificationServiceId: "" }
+    })
+  } as GlobalState;
+  const undefinedState = {
+    remoteConfig: O.some({
+      pn: { notificationServiceId: undefined }
+    })
+  } as GlobalState;
+
+  it("should return a serviceId when present, or undefined when missing", () => {
+    const resultSome = pnMessageServiceIdSelector(someState);
```

Comment:
Isn't it better to run a single unit tests with one input at a time, instead of four cases in just one test?